src/components/CompareResults/beta/ResultsTable.tsx

```diff
@@ -41,6 +41,8 @@ function processResults(results: CompareResultsItem[]) {
          test: entry[1][0].test,
          option_name: entry[1][0].option_name,
          extra_options: entry[1][0].extra_options,
+         new_rev: entry[1][0].new_rev,
+         new_repo: entry[1][0].new_repository_name,
```

Comment:
I guess this is already part of the results returned by the server?

Reply:
Yes, the changes were deployed last week.